packages/preload/src/index.ts

```diff
@@ -1887,6 +1888,13 @@ export function initExposure(): void {
     return ipcInvoke('kubernetes:getResourcesCount');
   });
 
+  contextBridge.exposeInMainWorld(
+    'kubernetesGetResources',
+    async (resourceName: string): Promise<KubernetesContextResources[]> => {
```

Comment:
question: does the frontend will call this method repeatably ? because this is a lot of serialization/deserialization, to provide all resources for all the contexts for every update